Library/Homebrew/cmd/audit.rb

```diff
@@ -363,10 +363,16 @@ def audit_deps
           problem "Don't use git as a dependency"
         when "mercurial"
           problem "Use `depends_on :hg` instead of `depends_on 'mercurial'`"
-        when "ruby"
-          problem "Don't use ruby as a dependency. We allow non-Homebrew ruby installations."
         when "gfortran"
           problem "Use `depends_on :fortran` instead of `depends_on 'gfortran'`"
+        when "ruby"
+          problem <<-EOS.undent
+            Don't use "ruby" as a dependency. If this formula requires a
+            minimum Ruby version not provided by the system you should
+            use the RubyRequirement:
```

Comment:
Some, probably. I wasn't overly keen on including the explicit `RubyRequirement` mention but we've been doing so for `open-mpi` for a while & nobody's publicly gone _"Huh?"_.